Packages/com.unity.inputsystem/Documentation~/UISupport.md

```diff
@@ -16,32 +16,27 @@ uid: input-system-ui-support
 
 
-⭐ From **Unity 2023.2.x** and newer, the Input System and UI Toolkit are fully integrated. To configure UI Input you can
-configure the UI action map of project-wide actions in **[Project Settings > Input System Package > Input Actions](Workflow-Actions.html)**.
-For older versions, the integration between the Input System and the UI system is handled by the [InputSystemUIInputModule](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html) component.
+From **Unity 2023.2** onwards, the Input System and UI Toolkit are fully integrated with each other. To configure UI Input you can configure the UI action map of [project-wide actions](Workflow-Actions.html) in **Project Settings > Input System Package > Input Actions**.
 
+For older versions of Unity, the integration between the Input System and the UI system is handled by the [Input System UI Input Module](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html) component.
 
->[!NOTE]
->The Input System package does not support IMGUI. If you have `OnGUI` methods in your player code (Editor code is unaffected), Unity does not receive any input events in those methods when the **Active Input Handling** [Player Setting](https://docs.unity3d.com/Manual/class-PlayerSettings.html) is set to **Input System Package**. To restore functionality you can change the setting to **Both**, but this means that Unity processes the input twice.
+
+> **Note:** The Input System package does not support [Immediate Mode GUI](https://docs.unity3d.com/Manual/GUIScriptingGuide.html) (IMGUI). If you have `OnGUI` methods in your player code (Editor code is unaffected), Unity does not receive any input events in those methods when the **Active Input Handling** [Player Setting](https://docs.unity3d.com/Manual/class-PlayerSettings.html) is set to **Input System Package**. To restore functionality you can change the setting to **Both**, but this means that Unity processes the input twice.
 
 ## Setting up UI input
 
-When using the project-wide actions (Unity 2023.2.x and newer), you can configure the UI action map of project-wide actions in **Project Settings > Input System Package > Input Actions)**.
+When using [project-wide actions](Workflow-Actions.html) in Unity 2023.2 and newer, you can configure the UI action map in **Project Settings > Input System Package > Input Actions**.
 
 ![ProjectSettingsInputActionsUIActionMap](Images/ProjectSettingsInputActionsUIActionMap.png)
 
+You can modify, add, or remove bindings to the named actions in the UI action map to suit your project, however in order to remain compatible with UI Toolkit, the name of the action map ("UI") and the names of the actions it contains ("Navigate", "Submit", "Cancel", etc) must remain the same. This is because the actions names match the action property names in the [UI Input Module](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html) class.
```

Comment:
This part isn't quite right - I think you can drop the ' This is because' part and it would be more correct.

 This is because the actions names match the action property names in the [UI Input Module](../api/UnityEngine.InputSystem.UI.InputSystemUIInputModule.html) class.

The are the same but that';s not what we are matching against.

Reply:
An attempt to modify this is coming in pending commit.